assets/js/phoenix_live_view.js

```diff
@@ -1686,14 +1686,14 @@ export class View {
     this.attachTrueDocEl()
     let patch = new DOMPatch(this, this.el, this.id, html, null)
     patch.markPrunableContentForRemoval()
-    this.joinPending = false
-    this.performPatch(patch)
+    this.performPatch(patch, false)
     this.joinNewChildren()
     DOM.all(this.el, `[${this.binding(PHX_HOOK)}]`, hookEl => {
       let hook = this.addHook(hookEl)
       if(hook){ hook.__trigger__("mounted") }
     })
 
+    this.joinPending = false
```

Comment:
I have moved this closer to the `applyPendingUpdates` to avoid events arriving before we apply the pending ones.